pkg/note/note.go

```diff
@@ -93,67 +93,70 @@ func isValidName(name string) bool {
 }
 
 // genConformantKeyHash generates a 4-byte key ID for typical (non-ECDSA) keys.
```

Comment:
update comment for return type? to match `KeyHash`. Although there are all internal, maybe not relevant.

Reply:
This comment is accurate, this only returns a 4-byte key ID for a non-ECDSA key.

Reply:
it's returning (uint32, []byte) which is 4bytes+sum?

Reply:
Oh sorry, yep, misread this. Updating the comment